### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kaurov**
mention .htaccess

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ cd ./my-project
 ./vendor/bin/pimcore-install
 ```
 
-- Point your virtual host to `my-project/public` 
+- Point your virtual host to `my-project/public`
+- Create `my-project/public/.htaccess` according to https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/Apache_Configuration/
```
Comment (jdreesen) [inline README.md]: The skeleton uses nginx as its webserver and nginx doesn't need/understand `.htaccess` files.

Comment (dvesh3) [reply README.md]:
```suggestion
- [Only for Apache] Create `my-project/public/.htaccess` according to https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/Apache_Configuration/ 
```

**Comment by dvesh3** [comment]
@kaurov please sign CLA so that we can merge your changes :blush:

**Comment by kaurov** [comment]
> @kaurov please sign CLA so that we can merge your changes 😊

done

**Comment by dvesh3** [comment]
@kaurov thanks!

**Pull request merged**